### ISSUE TITLE
skimage.viewer: _cascade module not found

### ISSUE THREAD

**Issue by AaronStiff** [scikit-image/scikit-image#6267]
## Description

When importing skimage.viewer I get the following error:

```
Traceback (most recent call last):
  File "C:\Users\Aaron\Code\Python\Scripts\MyScripts\testing.py", line 3, in <module>
    import skimage.viewer
  File "C:\Users\Aaron\Code\Python\lib\site-packages\skimage\viewer\__init__.py", line 9, in <module>
    from .viewers import ImageViewer, CollectionViewer
  File "C:\Users\Aaron\Code\Python\lib\site-packages\skimage\viewer\viewers\__init__.py", line 1, in <module>
    from .core import ImageViewer, CollectionViewer
  File "C:\Users\Aaron\Code\Python\lib\site-packages\skimage\viewer\viewers\core.py", line 14, in <module>
    from ..plugins.base import Plugin
  File "C:\Users\Aaron\Code\Python\lib\site-packages\skimage\viewer\plugins\__init__.py", line 2, in <module>
    from .canny import CannyPlugin
  File "C:\Users\Aaron\Code\Python\lib\site-packages\skimage\viewer\plugins\canny.py", line 4, in <module>
    from ...feature import canny
  File "C:\Users\Aaron\Code\Python\lib\site-packages\skimage\feature\__init__.py", line 4, in <module>
    from ._cascade import Cascade
ImportError: DLL load failed while importing _cascade: The specified module could not be found.
```

## Way to reproduce
```python
import skimage.viewer
```

## Version information
```python
3.10.2 (tags/v3.10.2:a58ebcc, Jan 17 2022, 14:12:15) [MSC v.1929 64 bit (AMD64)]
Windows-10-10.0.19044-SP0
scikit-image version: 0.19.2
numpy version: 1.22.2
```

**Comment by grlee77** [comment]
Thanks for reporting the issue. We will look into it. 

Just as a heads up, the viewer model itself is no longer being updated and is scheduled for removal in 1.0. We generally recommend [napari]](https://napari.org) as a more actively developed viewer with more features.

**Comment by hmaarrfk** [comment]
Ah, so you are able to use other feature of `skimage` as you needed?

**Comment by AaronStiff** [comment]
@hmaarrfk That is correct. However, I tested `from skimage.feature import Cascade` and I still get the same error.

**Comment by AaronStiff** [comment]
Installed scikit-image with pip, and downloaded Python from python.org. Not using any virtual envinronments, Anaconda, etc.

**Comment by svetlak0f** [comment]
Hi, i think i solved this problem. I have installed the packages C++ with Visual studion installer, and the error didn't reappear.

**Comment by Flintols** [comment]
loading scikit-image above 17.2 kills my python runner when using VS Code as well

**Comment by Bangik** [comment]
> Hi, i think i solved this problem. I have installed the packages C++ with Visual studio installer, and the error did not appear.

This is work for me
I download and install from this [Link](https://learn.microsoft.com/en-US/cpp/windows/latest-supported-vc-redist?view=msvc-170) and then restart my laptop. After that i solved the same problem

**Comment by hmaarrfk** [comment]
Sorry I didn't respond for a while. I can't speak to anybody's issues except for @AaronStiff's.

I believe your issue is that in the past you've used 

```
pip install --user [something else following this]
```
this causes serious issues as you update between different python environments because it installs a package (with compiled dependencies) specific to the version of python at time of installation, and then is used by future versions of python.

My suggestion would be to start python with the environment variable `PYTHONVERBOSE=1` before starting it. I'm not too sure how to do this in vscode, but if set, when I type

```
import skimage
```
a lot of output is spewed in the form of:

```
# /home/mark/mambaforge/envs/dev/lib/python3.9/site-packages/skimage/_shared/__pycache__/tester.cpython-39.pyc matches /home/mark/mambaforge/envs/dev/lib/python3.9/site-packages/skimage/_shared/tester.py
# code object from '/home/mark/mambaforge/envs/dev/lib/python3.9/site-packages/skimage/_shared/__pycache__/tester.cpython-39.pyc'
```

which gives you an indication of where things are installed.

I have a feeling that you will see, in your case, many different directories that have `site-packages` in them that are used.

I believe you should delete some of them. It is hard for me to tell you which ones since I don't know the details of which python VSCode is using exactly.

In our installation instructions, we really suggest using virtual environment 

https://scikit-image.org/docs/stable/install.html#install-via-pip 

As for the others that experienced similar issues, hopefully the troubleshooting information here can help you. Otherwise, please open a new issue. Some of the information on the template is critical for us helping you constructively.

In light of my late replies, I am going to assume that @AaronStiff has resolved their issue in the last year. I am going to close this issue. @AaronStiff if you still need help, please comment and I will gladly re-open. Sorry for not responding again back in February.
